infrastructure/gcp/keep-test/google-functions/keep-faucet/parse-account-address.js

```diff
@@ -3,18 +3,18 @@ const url = require('url')
 
 exports.parseAccountAddress = (request, response) => {
   requestUrl = url.parse(request.url, true)
```

Comment:
This is never declared. `const requestUrl = …`.

Reply:
oh my.